app/main.js

```diff
@@ -1,163 +1,229 @@
-const { app, BrowserWindow, ipcMain, shell } = require('electron');
+const electron = require('electron');
+const {
+    app,
+    BrowserWindow,
+    ipcMain, 
+    Menu, 
+    shell
+} = electron;
 const fs = require('fs-extra');
 const storage = require('electron-json-storage');
 const _ = require('lodash');
 const SWProxy = require('./proxy/SWProxy');
-
 const path = require('path');
 const url = require('url');
-
 global.gMapping = require('./mapping');
-
 global.win;
 let defaultFilePath = path.join(app.getPath('desktop'), app.getName() + ' Files');
+let template = [{
+  label: 'Window',
+  role: 'window',
+  submenu: [{
+    label: 'Minimize',
+    accelerator: 'CmdOrCtrl+M',
+    role: 'minimize'
+  }]
+}, {
+  label: 'Help',
+  role: 'help',
+  submenu: [{
+    label: "Summoner's War Exporter GitHub",
+    click: function () {
+      shell.openExternal('https://github.com/Xzandro/sw-exporter')
+    }
+  }]
+}]
+
+function findReopenMenuItem () {
```

Comment:
What is this function for? I see no usages and it is not something used by Electron as far as I know